site/layouts/shortcodes/scss-docs.html

```diff
@@ -41,7 +41,7 @@
 
   <div class="bd-example-snippet bd-code-snippet bd-scss-docs">
     <!--Boosted mod: added `border-light border-1 border-top-0`-->
```

Comment:
Haven't checked the change technically but this comment should be updated whatever the modification we validate